pkg/authorization/api/v1/register.go

```diff
@@ -45,6 +46,13 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&ClusterPolicyBindingList{},
 		&ClusterRoleBindingList{},
 		&ClusterRoleList{},
+
+		&RoleBindingRestriction{},
+		&RoleBindingRestrictionList{},
 	)
 	return nil
 }
+
+func (obj *RoleBindingRestrictionList) GetObjectKind() unversioned.ObjectKind { return &obj.TypeMeta }
```

Comment:
I expected you to get these automatically.  Can you delete them and have it still compile/work?